### PR TITLE
Add tutorials for training keypoint detection

### PR DESCRIPTION
**Pull request by yd8534976**
Preview: https://deploy-preview-685--blueoil.netlify.com/tutorial/image_keypoint_detection.html
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/usage/dataset.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Training data is required; validation data is optional. When you only provide tr
 - Task type: `Semantic Segmentation`
   - [CamvidCustom](#camvidcustom)
 
+- Task type: `Keypoint Detection`
+  - [MSCOCO_2017 keypoint detection](MSCOCO_2017-keypoint-detection)
```
Comment (iizukak) [inline docs/usage/dataset.md]: This link is missing

File: docs/tutorial/image_keypoint_detection.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+Training runs on the TensorFlow backend. So you can use TensorBoard to visualize your training process.
+
+    $ tensorboard --logdir=${OUTPUT_DIR}/keypoint_detection_demo_{TIMESTAMP} --port {Port}
+
+- input / labels_heatmap / output_heatmap
+<img src="../_static/keypoint_detection_heatmap.png">
+
+- visualize_labels, visualize_output
+<img src="../_static/keypoint_detection_visualize.png">
+
```
Comment (iizukak) [inline docs/tutorial/image_keypoint_detection.md]: It's better to add some metrics screenshots.

File: docs/usage/dataset.md (hunk 1 above)
Comment (yd8534976) [reply docs/usage/dataset.md]: Thanks! I have fixed it.

File: docs/tutorial/image_keypoint_detection.md (hunk 2 above)
Comment (yd8534976) [reply docs/tutorial/image_keypoint_detection.md]: Added.

**Comment by iizukak** [comment]
🤔

**Pull request merged**